### DISCUSSION TITLE
login credentials recover

### DISCUSSION THREAD

**Discussion by PatrizioRD** [mckaywrigley/chatbot-ui#1842] [Help]
I installed ChatBot-UI more than 1 year ago, while trying and testing around this and other local running solutions.
So, I literally didn't bother for a moment about storing the login credentials I was using for one of many installation I was trying on my own machine.
Today, after a crash and a recover, I found my local installation asking for login credentials and now I'm crying a river, after looking around for hours and failing to access the database.
I successfully created a new profile, and everything works as expected, but the old one had a lot of carefully ordered folders for old chats and a bunch of assistants and presets.
 
Can anyone point me in the right direction out from this situation I brilliantly created myself?

**Comment by Davitcoin** [comment]
First we need to check how you install your app, Do you use Docker, Local DB, or supabase? i´m pretty sure you erase the data from the DB

**Reply by PatrizioRD** [reply]
Thanks for your reply. I accurately followed the Local Quickstart instructions, so I'm running supabase locally with Docker.
